flake.nix

```
@@ -32,11 +32,25 @@
                 __structuredAttrs = true;
                 unsafeDiscardReferences.out = true;
               };
+
+          nix-index-small-database =
+            (pkgs.fetchurl {
+              url = generated.url + pkgs.stdenv.system + "-small";
+              hash = generated.hashes."${pkgs.stdenv.system}-small";
+            }).overrideAttrs
+              {
+                __structuredAttrs = true;
+                unsafeDiscardReferences.out = true;
+              };
         in
         {
-          inherit nix-index-database;
+          inherit nix-index-database nix-index-small-database;
 
           nix-index-with-db = pkgs.callPackage ./nix-index-wrapper.nix { inherit nix-index-database; };
+          nix-index-with-small-db = pkgs.callPackage ./nix-index-wrapper.nix {
+            nix-index-database = nix-index-small-database;
+            db-type = "small";
+          };
           comma-with-db = pkgs.callPackage ./comma-wrapper.nix { inherit nix-index-database; };
```

I think we want this for comma as well:

```suggestion
          comma-with-db = pkgs.callPackage ./comma-wrapper.nix { inherit nix-index-small-database; };
```